packages/flutter_test/lib/src/_matchers_io.dart

```diff
@@ -22,7 +22,7 @@ import 'goldens.dart';
 ///
 ///  * [OffsetLayer.toImage] which is the actual method being called.
 Future<ui.Image> captureImage(Element element) {
-  RenderObject renderObject = element.renderObject;
+  RenderObject renderObject = element.renderObject!;
```

Comment:
should we check if renderObject is non-null? Or is it more or less guaranteed to be?

Reply:
Added an assert.